combat-trainer.lic

```diff
@@ -566,7 +566,11 @@ class SpellProcess
     echo("  @offensive_spell_cycle: #{@offensive_spell_cycle}") if $debug_mode_ct
 
     @spell_data = get_data('spells').spell_data
-
+    @offensive_spells.each.with_index do |spell, i| 
```

Comment:
It looks like it can be done in three ways. The current version works too (http://apidock.com/ruby/Enumerator/each_with_index)